integration/common/tests/dbt/test_dbt_local.py

```diff
@@ -268,3 +268,88 @@ def test_build_target_path(test_name, dbt_project, expected):
         job_namespace="ol-namespace",
     )
     assert processor.build_target_path(dbt_project) == expected
+
+
+@pytest.mark.parametrize(
+    "path",
+    [
+        "tests/dbt/compiled_code",
+    ],
+)
+def test_dbt_column_lineage(path, parent_run_metadata):
+    """
+    Test that column-level lineage is correctly extracted from dbt artifacts
+    """
+    processor = DbtLocalArtifactProcessor(
+        producer="https://github.com/OpenLineage/OpenLineage/tree/0.0.1/integration/dbt",
+        job_namespace="job-namespace",
+        project_dir=path,
+    )
+    processor.dbt_run_metadata = parent_run_metadata
+    dbt_events = processor.parse()
+    events = [
+        attr.asdict(event, value_serializer=serialize)
+        for event in dbt_events.starts + dbt_events.completes + dbt_events.fails
+    ]
+    with open(f"{path}/result.json") as f:
+        assert match(json.load(f), events)
```

Comment:
@mayurmadnani there is no column-level lineage in the result event: https://github.com/OpenLineage/OpenLineage/blob/main/integration/common/tests/dbt/compiled_code/result.json

The comparison won't check existence of CLL in the event your change is generating.